src/main/java/com/flightstats/hub/dao/aws/S3AccessMonitor.java

```diff
@@ -89,6 +89,7 @@ private String key() {
                     return s3Object.getObjectMetadata().getVersionId();
                 } catch (IOException e) {
                     log.info("error closing connection to s3", e);
+                    return "";
```

Comment:
It would be nice to use constant here StringUtils.EMPTY or declare it in the class. This prevents the accidental modification of the "".